src/main/scala/com/campudus/tableaux/controller/MediaController.scala

```diff
@@ -209,12 +202,21 @@ class MediaController(override val config: TableauxConfig,
 
   def deleteFile(uuid: UUID): Future[TableauxFile] = {
     for {
+    // retrieve cells with this file for cache invalidation
+      cellsForFiles <- attachmentModel.retrieveCells(uuid)
```

Comment:
And after deletion of file we wouldn't find any attachment cell with this uuid. So, no we need to do this before deleting the file.